views/getJokes.ejs

```diff
@@ -1,24 +1,24 @@
 <% include partials/header %>
-<% var jokeType = data.type %>
 
 <div class="ui container">
     <div class="ui raised centered yellow card">
         <div class="content">
-            <% if(jokeType ==="single") { %>
+             <% if(data.urlName==="dadJokes") { %>
```

Comment:
this if-else statement assumes that there are only two types of jokes that can be returned(Dad Jokes and Chuck Norris). Instead, we will have more types of jokes. So use only if statement for each type of joke without using else.

Pseudocode:
```
if(isDadJoke) {
    // show the joke
}

if (isChuckNorrisJoke) {
    ...
}

if (anothTypeOfJoke) {
    ...
}
```